### DISCUSSION TITLE
머지 커밋 메시지 설명란 관련 의견 제안

### DISCUSSION THREAD

**Discussion by enyo9rt** [2022-Harmony/NewsCommunity-fFinal#47] [Ideas]
커밋 메시지를 https://github.com/2022-Harmony/NewsCommunity-fFinal/pull/45/commits/f9a2c973abe35b92d50b580f004e0070ea50d25f 이런 식으로 추가하는 것은 어떤가요?

@hyunzxn @choichanhyeok  @YoungeunJo

**Comment by hyunzxn** [comment]
그러면 그 머지할 때 커밋메시지를 다 넣는게 아니라 그 이상한 이슈 번호만 넣자는 말씀이져? 저는 좋은 것 같습니다.

**Reply by enyo9rt** [reply]
아 저거 이슈 번호가 아니라 커밋 번호예요!!!

**Reply by hyunzxn** [reply]
아 ㅋㅋㅋㅋ 그렇군요. 그럼 커밋 메시지를 옮겨적는게 아니라 커밋 번호를 넣자는거네요?

**Reply by enyo9rt** [reply]
네 저건 커서 올리면 커밋 내용도 뜨고 커밋 연결도 되고요!

**Reply by hyunzxn** [reply]
저는 좋은 것 같습니다. 다른 분들 생각도 들어보고 괜찮으면 적용하죠!

**Reply by choichanhyeok** [reply]
예 알겠습니다
